### ISSUE TITLE
BOM without Spring Boot dependencies?

### ISSUE THREAD

**Issue by Stephan202** [stormpath/stormpath-sdk-java#1220]
When importing `com.stormpath.sdk:stormpath-bom` instead of the individual Stormpath dependencies, all dependencies specified by `org.springframework.boot:spring-boot-dependencies` are transitively pulled in as well. Is there any way this can be avoided?

Our use case: we would like to use the new BOM, but our project imports Guava and enforces [dependency:analyze-dep-mgt](https://maven.apache.org/plugins/maven-dependency-plugin/analyze-dep-mgt-mojo.html). This is incompatible with importing `org.springframework.boot:spring-boot-dependencies`, because it yields the following section in the effective POM:

```xml
      <dependency>
        <groupId>com.datastax.cassandra</groupId>
        <artifactId>cassandra-driver-dse</artifactId>
        <version>2.1.9</version>
        <exclusions>
          <exclusion>
            <artifactId>slf4j-log4j12</artifactId>
            <groupId>org.slf4j</groupId>
          </exclusion>
          <exclusion>
            <artifactId>log4j</artifactId>
            <groupId>log4j</groupId>
          </exclusion>
          <exclusion>
            <artifactId>guava</artifactId>
            <groupId>com.google.guava</groupId>
          </exclusion>
        </exclusions>
      </dependency>
```

This then causes:
```
[INFO] --- maven-dependency-plugin:3.0.0:analyze-dep-mgt (analyze-dependencies) @ exception-support ---
[INFO] Found Resolved Dependency/DependencyManagement mismatches:
[INFO] com.google.guava:guava:jar was excluded in DepMgt, but version 21.0-rc2 has been found in the dependency tree.
```

**Comment by dogeared** [comment]
Hey @Stephan202 - Thanks for logging the issue. I am looking into this.

**Comment by Stephan202** [comment]
@dogeared, I'll close this issue now that you resolved #1221 and #1222. The issue does not reproduce on version 1.4.0.
